{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "n/a",
                    "vendor": "n/a",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "n/a"
                        }
                    ]
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "value": "A code injection issue was discovered in PyXDG before 0.26 via crafted Python code in a Category element of a Menu XML document in a .menu file. XDG_CONFIG_DIRS must be set up to trigger xdg.Menu.parse parsing within the directory containing this file. This is due to a lack of sanitization in xdg/Menu.py before an eval call."
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "n/a",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2021-08-03T14:06:14",
                "orgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
                "shortName": "mitre"
            },
            "references": [
                {
                    "tags": [
                        "x_refsource_MISC"
                    ],
                    "url": "https://gist.github.com/dhondta/b45cd41f4186110a354dc7272916feba"
                },
                {
                    "tags": [
                        "x_refsource_MISC"
                    ],
                    "url": "https://snyk.io/vuln/SNYK-PYTHON-PYXDG-174562"
                },
                {
                    "name": "[debian-lts-announce] 20190616 [SECURITY] [DLA 1819-1] pyxdg security update",
                    "tags": [
                        "mailing-list",
                        "x_refsource_MLIST"
                    ],
                    "url": "https://lists.debian.org/debian-lts-announce/2019/06/msg00006.html"
                },
                {
                    "name": "[debian-lts-announce] 20210803 [SECURITY] [DLA 2727-1] pyxdg security update",
                    "tags": [
                        "mailing-list",
                        "x_refsource_MLIST"
                    ],
                    "url": "https://lists.debian.org/debian-lts-announce/2021/08/msg00003.html"
                }
            ],
            "x_legacyV4Record": {
                "CVE_data_meta": {
                    "ASSIGNER": "cve@mitre.org",
                    "ID": "CVE-2019-12761",
                    "STATE": "PUBLIC"
                },
                "affects": {
                    "vendor": {
                        "vendor_data": [
                            {
                                "product": {
                                    "product_data": [
                                        {
                                            "product_name": "n/a",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "n/a"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                },
                                "vendor_name": "n/a"
                            }
                        ]
                    }
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "eng",
                            "value": "A code injection issue was discovered in PyXDG before 0.26 via crafted Python code in a Category element of a Menu XML document in a .menu file. XDG_CONFIG_DIRS must be set up to trigger xdg.Menu.parse parsing within the directory containing this file. This is due to a lack of sanitization in xdg/Menu.py before an eval call."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "eng",
                                    "value": "n/a"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "https://gist.github.com/dhondta/b45cd41f4186110a354dc7272916feba",
                            "refsource": "MISC",
                            "url": "https://gist.github.com/dhondta/b45cd41f4186110a354dc7272916feba"
                        },
                        {
                            "name": "https://snyk.io/vuln/SNYK-PYTHON-PYXDG-174562",
                            "refsource": "MISC",
                            "url": "https://snyk.io/vuln/SNYK-PYTHON-PYXDG-174562"
                        },
                        {
                            "name": "[debian-lts-announce] 20190616 [SECURITY] [DLA 1819-1] pyxdg security update",
                            "refsource": "MLIST",
                            "url": "https://lists.debian.org/debian-lts-announce/2019/06/msg00006.html"
                        },
                        {
                            "name": "[debian-lts-announce] 20210803 [SECURITY] [DLA 2727-1] pyxdg security update",
                            "refsource": "MLIST",
                            "url": "https://lists.debian.org/debian-lts-announce/2021/08/msg00003.html"
                        }
                    ]
                }
            }
        },
        "adp": [
            {
                "providerMetadata": {
                    "orgId": "96f1cf3c-f80a-4e0f-a063-99a91e3f1325",
                    "shortName": "secretariat-reference",
                    "dateUpdated": "2024-07-02T10:22:35.795Z"
                },
                "references": [
                    {
                        "tags": [
                            "x_refsource_MISC",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://gist.github.com/dhondta/b45cd41f4186110a354dc7272916feba/DEMO_2024_06_19"
                    },
                    {
                        "tags": [
                            "x_refsource_MISC",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://snyk.io/vuln/SNYK-PYTHON-PYXDG-174562/DEMO_2024_06_19"
                    },
                    {
                        "name": "[debian-lts-announce] 20190616 [SECURITY] [DLA 1819-1] pyxdg security update",
                        "tags": [
                            "mailing-list",
                            "x_refsource_MLIST",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://lists.debian.org/debian-lts-announce/2019/06/msg00006.html/DEMO_2024_06_19"
                    },
                    {
                        "name": "[debian-lts-announce] 20210803 [SECURITY] [DLA 2727-1] pyxdg security update",
                        "tags": [
                            "mailing-list",
                            "x_refsource_MLIST",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://lists.debian.org/debian-lts-announce/2021/08/msg00003.html/DEMO_2024_06_19"
                    },
                    {
                        "tags": [
                            "x_refsource_MISC",
                            "x_snapshot"
                        ],
                        "url": "https://gist.github.com/dhondta/b45cd41f4186110a354dc7272916feba/adp-test-6-28"
                    },
                    {
                        "tags": [
                            "x_refsource_MISC",
                            "x_snapshot"
                        ],
                        "url": "https://snyk.io/vuln/SNYK-PYTHON-PYXDG-174562/adp-test-6-28"
                    },
                    {
                        "name": "[debian-lts-announce] 20190616 [SECURITY] [DLA 1819-1] pyxdg security update",
                        "tags": [
                            "mailing-list",
                            "x_refsource_MLIST",
                            "x_snapshot"
                        ],
                        "url": "https://lists.debian.org/debian-lts-announce/2019/06/msg00006.html/adp-test-6-28"
                    },
                    {
                        "name": "[debian-lts-announce] 20210803 [SECURITY] [DLA 2727-1] pyxdg security update",
                        "tags": [
                            "mailing-list",
                            "x_refsource_MLIST",
                            "x_snapshot"
                        ],
                        "url": "https://lists.debian.org/debian-lts-announce/2021/08/msg00003.html/adp-test-6-28"
                    }
                ]
            }
        ]
    },
    "cveMetadata": {
        "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
        "assignerShortName": "mitre",
        "cveId": "CVE-2019-12761",
        "datePublished": "2019-06-06T18:55:27",
        "dateReserved": "2019-06-06T00:00:00",
        "dateUpdated": "2024-07-02T10:22:35.795Z",
        "state": "PUBLISHED"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1"
}